tests/pytests/unit/utils/test_network.py

```diff
@@ -1557,6 +1561,7 @@ def test_interface_and_ip(linux_interfaces_dict):
         assert ret == 'Interface "dog" not in available interfaces: "eth0", "lo"'
 
 
+@pytest.mark.skip_on_windows(reason="Do not run on Windows")
```

Comment:
Can't this be updated to make the test work on windows too?

Reply:
Sure, let me see if I can figure that out.